scripts/metricsLib/metrics_definitions.py

```diff
@@ -116,6 +119,16 @@
                                  {"total_project_blank_lines": ["blank_lines"],
                                  "average_blank_lines": ["avg_blank_lines"]}))
 
+SIMPLE_METRICS.append(ListMetric("repositoryLanguages", 
```

Comment:
A `ListMetric` won't parse this endpoint properly because the data returned is not returned inside a list. 

You will either need to use a `CustomMetric` or create your own class to parse this endpoint since the data returned is a dictionary that has inconsistent keys. Meaning, you won't always see the same keys returned in the JSON data consistently. 

I recommend creating your own subclass inside `metrics_data_structures.py` that inherits from `BaseMetric` and overwrites the `get_values` method. This way you can use an object of this class to parse the endpoint and any endpoint like it in the future. 

The data returned is parsed inside the metric's `get_values` method. You can look at the other classes such as `LengthMetric` that have a version of the method as an example.